### DISCUSSION TITLE
Access to Global semantic embeddings

### DISCUSSION THREAD

**Discussion by bmuhwezi** [Clay-foundation/model#325] [General]
Hi everyone,

I’ve recently started learning about the Clay Foundation model and had a quick question. I noticed that there are a few sample embeddings available on Source Cooperative, but I was wondering if there’s a global dataset available as well.

The reason I’m asking is that I’m looking to extract embeddings on a larger, global scale for use as input features in a model I’m currently training. Before I proceed with extracting the embeddings myself, I wanted to check if more extensive data is available.

Thanks in advance for your help!

Best regards,
Bob

**Comment by lauracchen** [comment]
Hi @bmuhwezi thanks for reaching out! 

We are actually going to run inference very soon on global Sentinel 2 composites - feel free to see details/weigh in [here](https://github.com/Clay-foundation/model/issues/277). We'll do so as soon as [Clay model v1.5 is done training](https://github.com/Clay-foundation/model/issues/283), which should be (conservatively) within the next couple of weeks.

Hope this helps!

Thanks,
Laura

**Reply by bmuhwezi** [reply]
Thanks for the info @lauracchen.
